### ISSUE TITLE
this.x(...).y is not a function

### ISSUE THREAD

**Issue by xorb** [opendesigndev/open-design-sdk#22]
## Details
-- yarn 1.22.17
-- node v14.18.3
-- @opendesign/sdk: "^0.4.2"
-- typescript: "^4.5.5"

## Sample code 
```ts
import { createSdk } from "@opendesign/sdk";

async function main() {
  try {
    const sdk = createSdk({
      token:"token"
    });

    // Open a local design file
    const design = await sdk.importDesignFile("./sample.ai");

    // Get the first artboard in the design
    const artboard = design.getArtboards()[0];
    console.log(artboard);
    // Get the first root layer
    const layers = await artboard.getRootLayers();
    const layer = layers[0];
    // Render the layer
    await layer.exportToSvgFile("layer.svg");
  } catch (err: any) {
    console.log(err.message);
  }
}

main();
```
## Output

API: GET https://illustr.../64c92a52-49eb-4bb0-9181-1289437d9267.png -> 200 OK
API: GET https://illustr.../641f3d22-00ca-4ac0-b605-3a9a3c394c29.png ...
API: GET https://illustr.../641f3d22-00ca-4ac0-b605-3a9a3c394c29.png -> 200 OK
API: GET https://illustr.../864a2d4c-8eb4-46a4-90f9-e6a92292bd91.png ...
API: GET https://illustr.../864a2d4c-8eb4-46a4-90f9-e6a92292bd91.png -> 200 OK
API: GET https://illustr.../5c4efcc2-2dba-481f-86b8-b8a56bde260b.png ...
API: GET https://illustr.../5c4efcc2-2dba-481f-86b8-b8a56bde260b.png -> 200 OK
this.x(...).y is not a function

**Comment by rhefner1** [comment]
Err, that doesn't look good. @petrbrzek can you please take a look at this?

**Comment by xorb** [comment]
Let me know if you need additional details on this.

**Comment by janroztocil** [comment]
Fixed in `0.4.5`.

**Comment by rhefner1** [comment]
@xorb let me know if this fixes the issue! :)
